javascript/chronometer.js

```diff
@@ -1,34 +1,40 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime=0;
+    this.intervalId=null;
   }
 
   start(callback) {
-    // ... your code goes here
+    const intervalId = setInterval(() => {this.currentTime +=1}, 1000); 
```

Comment:
you should have also checked for callback to be defined and executed callback. other than that, good use of an arrow function to preserve the `this` context and access instance's value

Reply:
also, the fact that you do not assign the interval's return value (intervalId) to the instance (by using `this.intervalId`, makes other tests fail (for example the stop function)